src/Dijkstra/AST/Maybe.agda

```diff
@@ -4,6 +4,10 @@
    Licensed under the Universal Permissive License v 1.0 as shown at https://opensource.oracle.com/licenses/upl
 -}
 
+-- TEMPORARY!!
+{-# OPTIONS --allow-unsolved-metas #-}
```

Comment:
Since `obm-dangerous-magic` is being used, I do think this is necessary.

Reply:
Yes, but TEMPORARY!!